src/ShopBundle/objects/db/TShopArticle.class.php

```diff
@@ -1907,6 +1907,8 @@ public function UpdateStock($dNewStockValue, $bNewAmountIsDelta = false, $bUpdat
             return false;
         }
         $oldStock = $this->getAvailableStock();
+        //$valueChanged = (double)$oldStock !== (double)$dNewStockValue;
```

Comment:
Please remove dead code.

Reply:
It's not really dead but "would be correct if desired".

Reply:
Never compare floating point values using exact comparison :-) - I think the note below this line suffices to tell the reader that there is room for improvement (and doesn't imply a solution that we yet need to think about).